### ISSUE TITLE
regression when building mega_bezel

### ISSUE THREAD

**Issue by chyyran** [SnowflakePowered/librashader#52]
```
called `Result::unwrap()` on an `Err` value: ShaderCompileError(GlslangError(ParseError("ERROR: params-0-screen-scale.inc:328: '' : numeric literal too big \nERROR: params-0-screen-scale.inc:328: '#pragma' : directive must end with a newline \nERROR: 2 compilation errors.  No code generated.\n\n")))
```

**Comment by chyyran** [comment]
Fixed by stripping `#pragma parameter`, but naga doesn't work because of `OriginalFeedbackSize`. SPIRV-Cross seems to strip this, we need to remove it in the naga reflect pass.

**Comment by chyyran** [comment]
fixed by #53 

no mega bezel for wgpu, the generated spirv is too big for the driver
